core/App/components/animated/LoadingIndicator.tsx

```diff
@@ -34,7 +34,7 @@ const LoadingIndicator: React.FC = () => {
 
   return (
     <View style={{ alignItems: 'center', justifyContent: 'center' }} testID={testIdWithKey('LoadingActivityIndicator')}>
-      <Image source={Assets.img.logoLarge.src} testID={testIdWithKey('LoadingActivityIndicatorX')} />
+      <Image source={Assets.img.logoSecondary.src} testID={testIdWithKey('LoadingActivityIndicatorX')} />
```

Comment:
Nit: can we make this `logoPrimary` since it is used in more than one screen compared to `logoSecondary`?